config.toml

```diff
@@ -111,6 +111,38 @@ name = "Community"
 url = "https://github.com/helm/community"
 weight = 5
 
+# Korean
+# [languages.ko]
+# title = "Helm"
+# description = "Helm - The Kubernetes Package Manager."
+# contentDir = "content/ko"
+# languageName = "한국어 Korean"
+# weight = 1
+
+# [[languages.ko.menus.main]]
+# name = "Home"
+# url = "/"
+# weight = 1
+
+# [[languages.ko.menus.main]]
+# name = "Docs"
+# url = "/docs"
+# weight = 2
+
+# [[languages.ko.menus.main]]
+# name = "Charts"
+# url = "https://hub.helm.sh/"
+# weight = 3
+# external = "yes"
+
+#[[languages.zko.menus.main]]
+#name = "Blog"
+#url = "/blog"
+#weight = 2
+
+# [languages.ko.params]
+# language_alternatives = ["en"]
+
 # Chinese
 # [languages.zh]
```

Comment:
Just wondering can `zh` be used standalone or does it need to be specific e.g. `zh-cn`?

Reply:
That's a great question. I dug into that when I wrote some docs on localization recently. You can find those at https://helm.sh/docs/community/localization/. Basically, we use the two letter language code in the current setup. Not sure what that does for things like the color vs colour debate.

Reply:
Might be worth investigating further to be sure.  It is a bit since I did localization but there is something about languages with colloquial derivations.  I think it might default `zh` to `zh-cn` in most code languages.